examples/Inflector/app/inflector_view_controller.rb

```diff
@@ -36,73 +36,74 @@ def update_fields(original)
+                                          :title => "Select",
+                                          :type => :button,
+                                          :key => :select
+                                        }]
+                                      }, {
+                                        :rows => [{
+                                          :title => "Inflect",
+                                          :type => :submit
+                                        }]
+                                      }, {
+                                        :title => "Inflections",
+                                        :rows => [{
+                                          :title => "Singular",
+                                          :key => :singular,
+                                          :type => :static
+                                        }, {
+                                          :title => "Plural",
+                                          :key => :plural,
+                                          :type => :static
+                                        }]
+                                      }, {
+                                        :title => "Transformations",
+                                        :rows => [{
+                                          :title => "Camelized",
+                                          :key => :camelized,
+                                          :type => :static
+                                        }, {
+                                          :title => "Lower-Camelized",
+                                          :key => :lower_camelized,
+                                          :type => :static
+                                        }, {
+                                          :title => "Underscored",
+                                          :key => :underscored,
+                                          :type => :static
+                                        }, {
+                                          :title => "Classified",
+                                          :key => :classified,
+                                          :type => :static
+                                        }, {
+                                          :title => "Dasherized",
+                                          :key => :dasherized,
+                                          :type => :static
+                                        }, {
+                                          :title => "Titleized",
+                                          :key => :titleized,
+                                          :type => :static
+                                        }, {
+                                          :title => "Humanized",
+                                          :key => :humanized,
+                                          :type => :static
+                                        }]
+                                      }])
```

Comment:
I don't think this indentation is particularly pretty, because it wastes so much horizontal space.